<?xml version="1.0" encoding="UTF-8" ?>
<testsuite tests="40" failures="13" name="io.cloudboost.CloudTableTest" time="12.258" errors="12" skipped="0">
  <properties>
    <property name="java.runtime.name" value="Java(TM) SE Runtime Environment"/>
    <property name="sun.boot.library.path" value="C:\Program Files\jdk1.8.0_05\jre\bin"/>
    <property name="java.vm.version" value="25.5-b02"/>
    <property name="java.vm.vendor" value="Oracle Corporation"/>
    <property name="maven.multiModuleProjectDirectory" value="E:\Engineer\programming\projects\javaSDK\git_repo"/>
    <property name="java.vendor.url" value="http://java.oracle.com/"/>
    <property name="path.separator" value=";"/>
    <property name="guice.disable.misplaced.annotation.check" value="true"/>
    <property name="java.vm.name" value="Java HotSpot(TM) Client VM"/>
    <property name="file.encoding.pkg" value="sun.io"/>
    <property name="user.script" value=""/>
    <property name="user.country" value="US"/>
    <property name="sun.java.launcher" value="SUN_STANDARD"/>
    <property name="sun.os.patch.level" value=""/>
    <property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
    <property name="user.dir" value="E:\Engineer\programming\projects\javaSDK\git_repo"/>
    <property name="java.runtime.version" value="1.8.0_05-b13"/>
    <property name="java.awt.graphicsenv" value="sun.awt.Win32GraphicsEnvironment"/>
    <property name="java.endorsed.dirs" value="C:\Program Files\jdk1.8.0_05\jre\lib\endorsed"/>
    <property name="os.arch" value="x86"/>
    <property name="java.io.tmpdir" value="C:\Users\new\AppData\Local\Temp\"/>
    <property name="line.separator" value="
"/>
    <property name="java.vm.specification.vendor" value="Oracle Corporation"/>
    <property name="user.variant" value=""/>
    <property name="os.name" value="Windows 8.1"/>
    <property name="classworlds.conf" value="C:\Program Files (x86)\apache-maven-3.3.3\bin\..\bin\m2.conf"/>
    <property name="sun.jnu.encoding" value="Cp1252"/>
    <property name="java.library.path" value="C:\Program Files\jdk1.8.0_05\bin;C:\Windows\Sun\Java\bin;C:\Windows\system32;C:\Windows;C:\ProgramData\Oracle\Java\javapath;C:\oraclexe\app\oracle\product\11.2.0\server\bin;;C:\Program Files (x86)\Intel\iCLS Client\;C:\Program Files\Intel\iCLS Client\;C:\Windows\system32;C:\Windows;C:\Windows\System32\Wbem;C:\Windows\System32\WindowsPowerShell\v1.0\;C:\Program Files\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\DAL;C:\Program Files (x86)\Intel\Intel(R) Management Engine Components\IPT;C:\Program Files\Intel\WiFi\bin\;C:\Program Files\Common Files\Intel\WirelessCommon\;C:\Program Files (x86)\GNU\GnuPG\pub;C:\Program Files (x86)\Microsoft SQL Server\100\Tools\Binn\;C:\Program Files\Microsoft SQL Server\100\Tools\Binn\;C:\Program Files\Microsoft SQL Server\100\DTS\Binn\;C:\Program Files (x86)\Skype\Phone\;C:\Program Files\nodejs\;C:\Program Files\MySQL\MySQL Server 5.6\bin;C:\Program Files (x86)\apache-ant-1.9.6\bin;C:\Program Files\jdk1.8.0_05\bin;E:\Engineer\eclipse\android-sdk-windows\tools;E:\Engineer\eclipse\android-sdk-windows\platform-tools;C:\Program Files\jdk1.8.0_05\bin;C:\Program Files\MongoDB\Server\3.0\bin;E:\Engineer\eclipse\eclipse;E:\Engineer\eclipse\myeclipse;C:\Program Files (x86)\apache-maven-3.3.3\bin;C:\Program Files (x86)\GNU\GnuPG;C:\Program Files\openssl-0.9.8k_X64\bin;C:\Program Files\nodejs;C:\Users\new\AppData\Roaming\npm;C:\Program Files\Wireshark;."/>
    <property name="java.specification.name" value="Java Platform API Specification"/>
    <property name="java.class.version" value="52.0"/>
    <property name="sun.management.compiler" value="HotSpot Client Compiler"/>
    <property name="os.version" value="6.3"/>
    <property name="user.home" value="C:\Users\new"/>
    <property name="user.timezone" value="Africa/Nairobi"/>
    <property name="java.awt.printerjob" value="sun.awt.windows.WPrinterJob"/>
    <property name="java.specification.version" value="1.8"/>
    <property name="file.encoding" value="Cp1252"/>
    <property name="user.name" value="new"/>
    <property name="java.class.path" value="C:\Program Files (x86)\apache-maven-3.3.3\bin\..\boot\plexus-classworlds-2.5.2.jar"/>
    <property name="java.vm.specification.version" value="1.8"/>
    <property name="sun.arch.data.model" value="32"/>
    <property name="java.home" value="C:\Program Files\jdk1.8.0_05\jre"/>
    <property name="sun.java.command" value="org.codehaus.plexus.classworlds.launcher.Launcher test"/>
    <property name="java.specification.vendor" value="Oracle Corporation"/>
    <property name="user.language" value="en"/>
    <property name="awt.toolkit" value="sun.awt.windows.WToolkit"/>
    <property name="java.vm.info" value="mixed mode"/>
    <property name="java.version" value="1.8.0_05"/>
    <property name="java.ext.dirs" value="C:\Program Files\jdk1.8.0_05\jre\lib\ext;C:\Windows\Sun\Java\lib\ext"/>
    <property name="sun.boot.class.path" value="C:\Program Files\jdk1.8.0_05\jre\lib\resources.jar;C:\Program Files\jdk1.8.0_05\jre\lib\rt.jar;C:\Program Files\jdk1.8.0_05\jre\lib\sunrsasign.jar;C:\Program Files\jdk1.8.0_05\jre\lib\jsse.jar;C:\Program Files\jdk1.8.0_05\jre\lib\jce.jar;C:\Program Files\jdk1.8.0_05\jre\lib\charsets.jar;C:\Program Files\jdk1.8.0_05\jre\lib\jfr.jar;C:\Program Files\jdk1.8.0_05\jre\classes"/>
    <property name="sun.stderr.encoding" value="cp437"/>
    <property name="java.vendor" value="Oracle Corporation"/>
    <property name="maven.home" value="C:\Program Files (x86)\apache-maven-3.3.3\bin\.."/>
    <property name="file.separator" value="\"/>
    <property name="java.vendor.url.bug" value="http://bugreport.sun.com/bugreport/"/>
    <property name="sun.cpu.endian" value="little"/>
    <property name="sun.io.unicode.encoding" value="UnicodeLittle"/>
    <property name="sun.stdout.encoding" value="cp437"/>
    <property name="sun.desktop" value="windows"/>
    <property name="sun.cpu.isalist" value="pentium_pro+mmx pentium_pro pentium+mmx pentium i486 i386 i86"/>
  </properties>
  <testcase classname="io.cloudboost.CloudTableTest" name="getTable" time="0.393">
    <failure message="Cannot PUT /bengi123/table/axgweyni" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/axgweyni
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$6.done(CloudTableTest.java:296)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.getTable(CloudTableTest.java:290)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="addColumnAfterSave" time="0.4"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldGiveSpecificTable" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotRenameColumn" time="0.399">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$21.done(CloudTableTest.java:841)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldNotRenameColumn(CloudTableTest.java:836)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldGetColumnFromTable" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="deleteCompanyTable" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="updateNewColumnIntoTable" time="1.171">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.Column.setRelatedTo(Column.java:181)
	at io.cloudboost.CloudTableTest$9$1$1.done(CloudTableTest.java:378)
	at io.cloudboost.CloudTable.get(CloudTable.java:336)
	at io.cloudboost.CloudTableTest$9$1.done(CloudTableTest.java:372)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest$9.done(CloudTableTest.java:368)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.updateNewColumnIntoTable(CloudTableTest.java:364)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="updateCompanySchema" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="deleteAddressTable" time="0.002"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="createCompanyTable" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="deleteEmployeeTable" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="createAndDeleteTable" time="0.445">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$5.done(CloudTableTest.java:265)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createAndDeleteTable(CloudTableTest.java:261)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="createTableAllDataTypes" time="0.51">
    <failure message="Cannot PUT /bengi123/table/xrsqvivr" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/xrsqvivr
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$16.done(CloudTableTest.java:622)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createTableAllDataTypes(CloudTableTest.java:617)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldGetAllTable" time="0.368">
    <failure message="Not Found" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Not Found
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$7.done(CloudTableTest.java:328)
	at io.cloudboost.CloudTable.getAll(CloudTable.java:298)
	at io.cloudboost.CloudTableTest.shouldGetAllTable(CloudTableTest.java:322)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldGiveAllTable" time="0"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="createTableStoreGeoPoint" time="0.419">
    <failure message="Cannot PUT /bengi123/table/nosmbprn" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/nosmbprn
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$26.done(CloudTableTest.java:1013)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createTableStoreGeoPoint(CloudTableTest.java:1008)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="createEmployeeTable" time="0"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="addColumnToExistingTable" time="0.564">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$13.done(CloudTableTest.java:515)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.addColumnToExistingTable(CloudTableTest.java:509)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="createAddressTable" time="0"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="updateEmployeeSchema" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="createTableWithTextListColumnRelation" time="0.497">
    <failure message="Cannot PUT /bengi123/table/textlistrelation" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/textlistrelation
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$28.done(CloudTableTest.java:1090)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createTableWithTextListColumnRelation(CloudTableTest.java:1085)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="createTableSetRelationsToExistingTables" time="0.608">
    <failure message="Cannot PUT /bengi123/table/kqdynczj" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/kqdynczj
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$27.done(CloudTableTest.java:1055)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createTableSetRelationsToExistingTables(CloudTableTest.java:1050)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotPassStringToUpdateColumn" time="0"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="sequentialTests" time="0.472">
    <failure message="Cannot PUT /bengi123/table/eoewjftg" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/eoewjftg
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$1.done(CloudTableTest.java:31)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.sequentialTests(CloudTableTest.java:25)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="GetAllTable" time="0.4">
    <failure message="Not Found" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Not Found
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$8.done(CloudTableTest.java:347)
	at io.cloudboost.CloudTable.getAll(CloudTable.java:298)
	at io.cloudboost.CloudTableTest.GetAllTable(CloudTableTest.java:341)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="setTableType" time="0.34">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$12.done(CloudTableTest.java:483)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.setTableType(CloudTableTest.java:478)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="renameTableFails" time="0.348">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$14.done(CloudTableTest.java:541)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.renameTableFails(CloudTableTest.java:537)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="createListTable" time="0.348">
    <failure message="Cannot PUT /bengi123/table/devjdyfg" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/devjdyfg
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$15.done(CloudTableTest.java:574)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createListTable(CloudTableTest.java:568)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="duplicateTable" time="0.682"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotChangeUniquePropertyOfPreDefinedColumn" time="0.357">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$22.done(CloudTableTest.java:873)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldNotChangeUniquePropertyOfPreDefinedColumn(CloudTableTest.java:868)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotChangeUniquePropertyOfDefaultColumn" time="0.34">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$23.done(CloudTableTest.java:906)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldNotChangeUniquePropertyOfDefaultColumn(CloudTableTest.java:902)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotChangeRequiredPropertyOfDefaultColumn" time="0.344">
    <failure message="Cannot PUT /bengi123/table/yivoqcxg" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/yivoqcxg
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$24.done(CloudTableTest.java:938)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldNotChangeRequiredPropertyOfDefaultColumn(CloudTableTest.java:932)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldChangeRequiredPropertyOfUserDefinedColumn" time="0.616">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$25.done(CloudTableTest.java:981)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldChangeRequiredPropertyOfUserDefinedColumn(CloudTableTest.java:975)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldCreateAndDeleteTable" time="0.371">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$10.done(CloudTableTest.java:425)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldCreateAndDeleteTable(CloudTableTest.java:421)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldCreateColumnThenDelete" time="0.368">
    <failure message="Cannot PUT /bengi123/table/rmjksatp" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/rmjksatp
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$18.done(CloudTableTest.java:695)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldCreateColumnThenDelete(CloudTableTest.java:689)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldUpdateColumnInTable" time="0.001"/>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotDeleteDefaultColumn" time="0.339">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$19.done(CloudTableTest.java:760)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldNotDeleteDefaultColumn(CloudTableTest.java:754)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="createTableSetRelationToDataType" time="0.42">
    <failure message="Cannot PUT /bengi123/table/mydatatyperelation" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/mydatatyperelation
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$29.done(CloudTableTest.java:1126)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.createTableSetRelationToDataType(CloudTableTest.java:1121)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldNotChangeDataTypeOfColumn" time="0.34">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at io.cloudboost.CloudTableTest$20.done(CloudTableTest.java:792)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldNotChangeDataTypeOfColumn(CloudTableTest.java:787)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</error>
  </testcase>
  <testcase classname="io.cloudboost.CloudTableTest" name="shouldGiveSpecificTableByName" time="0.389">
    <failure message="Cannot PUT /bengi123/table/btllparc" type="junit.framework.AssertionFailedError">junit.framework.AssertionFailedError: Cannot PUT /bengi123/table/btllparc
	at junit.framework.Assert.fail(Assert.java:47)
	at io.cloudboost.CloudTableTest$17.done(CloudTableTest.java:664)
	at io.cloudboost.CloudTable.save(CloudTable.java:371)
	at io.cloudboost.CloudTableTest.shouldGiveSpecificTableByName(CloudTableTest.java:658)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:44)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:15)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:41)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:20)
	at org.junit.internal.runners.statements.FailOnTimeout$1.run(FailOnTimeout.java:28)
</failure>
  </testcase>
</testsuite>